tests/Symfony/Validator/Metadata/Property/Restriction/PropertySchemaCollectionRestrictionTest.php

```diff
@@ -101,11 +101,11 @@ public function createProvider(): \Generator
             ]),
         ];
         $properties = [
-            'name' => [],
+            'name' => new \ArrayObject(),
```

Comment:
I don't understand why this changes

Reply:
Because we return now an ArrayObject instead of an array : https://github.com/api-platform/core/pull/5505/files#diff-49885c8dd6111c2324083b2b63a1e3d201c48d45b8c69c893b27e552166a8474R57